### ISSUE TITLE
`test/conftest.py` missing in 2.7.post0 sdist

### ISSUE THREAD

**Issue by mgorny** [ICRAR/crc32c#53]
I'm sorry for not looking closer while reporting the previous bug but it seems that sdist is also missing `conftest.py`. It has only:

```
crc32c-2.7.post0/test/test_benchmark.py
crc32c-2.7.post0/test/test_crc32c.py
```

so the tests fail:

```pytb
=============================================================== ERRORS ================================================================
________________________________________________ ERROR at setup of test_software_mode _________________________________________________
file /tmp/portage/dev-python/crc32c-2.7_p0/work/crc32c-2.7.post0/test/test_crc32c.py, line 35
  def test_software_mode(crc32c_is_available: bool) -> None:
E       fixture 'crc32c_is_available' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, reco
rd_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/tmp/portage/dev-python/crc32c-2.7_p0/work/crc32c-2.7.post0/test/test_crc32c.py:35
```

**Comment by rtobar** [comment]
@mgorny no problem, thanks again for noticing. I've fixed it, merged, and pushed `v2.7.post1` so a new release should be available shortly in PyPI. I've also updated our GHA to check that the source distribution is not only installable, but that tests can be run from it too, which should prevent this from happening again.

**Comment by mgorny** [comment]
Thanks. I can confirm that `.post1` seems to be good.
